### DISCUSSION TITLE
[macOS] Allow to choose between Cmd and Ctrl for shortcuts modifiers

### DISCUSSION THREAD

**Discussion by p10tr3k** [doublecmd/doublecmd#1716] [Suggestions]
Currently, all shortcuts start with Ctrl. It would be wonderful if there were a single configuration switch to change all shortcuts on macOS to use Cmd instead of Ctrl.

**Comment by rich2014** [comment]
simply providing a swap will cause a lot of key conflicts.

**Comment by lprot** [comment]
> Currently, all shortcuts start with Ctrl. It would be wonderful if there were a single configuration switch to change all shortcuts on macOS to use Cmd instead of Ctrl.

Default shortcuts config in DC for MacOS leads to many issues. I have reconfigured all the shortcuts in the way we have no confilcts with the keys reserved by MacOS and provided updated file here https://github.com/doublecmd/doublecmd/issues/1651

@alexx2000 @rich2014 not sure why we cannot just include the corrected file into dmg distro to make life of MacOS users easier?

**Comment by NN---** [comment]
Agree, the default shortucts for mac are not so useful without either changing mac default shortcuts or DC shortucts.
For instance Ctrl+Arrow in mac does totally different things, I don't need it, so I removed in mac.
However I configured Option+Number for quick shortcuts which conflicts with DC tabs, so had to change in DC to Cmd+Option+Number.
Same with Cmd+Tab for going to next tab instead of expected Ctrl+Tab. In mac Cmd+Tab is already a default shortcut for scrolling programs, so changed to Cmd+Option+Tab. It cannot be Option+Tab because Alt-Tab is too common to use its shortcut.
